modules/sandboxed-containers-deleting-kata-resources.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-kata-resources_{context}"]
+= Deleting {sandboxed-containers-first} Kata resources
+
+You can remove and uninstall the `kata` runtime and all its related resources, such as CRI-O config and `RuntimeClass`, from your cluster.
+
+.Prerequisites
+
+* You have {sandboxed-containers-first} 1.0.0 installed.
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the `KataConfig` custom resource by running the following command:
++
+[source,terminal]
+----
+oc delete kataconfig <KataConfig_CR_Name>
+----
+
+The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster. After you run the command above, your cluster is restored to the state prior to the installation process.
```

Comment:
```suggestion
The {sandboxed-containers-operator} removes all resources that were initially created to enable the runtime on your cluster. After you run the preceding command, your cluster is restored to the state that it was prior to the installation process.
```

**above** _adj_
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”